src/__tests__/to-have-style.js

```diff
@@ -116,6 +119,33 @@ describe('.toHaveStyle', () => {
     )
   })
 
+  test('handles inline customer properties', () => {
+    const {queryByTestId} = render(`
+    <span data-testid="color-example" style="--color: blue">Hello World</span>
+  `)
+    expect(queryByTestId('color-example')).toHaveStyle('--color: blue')
+  })
+
+  test('handles global customer properties', () => {
```

Comment:
Same comment about the use of the word "customer". Again, not sure if it's really meant to be that way. Let me know.

```suggestion
  test('handles global custom properties', () => {
```